src/app/loginComponents/accounts/external/accounts.component.ts

```diff
@@ -84,6 +84,14 @@ export class AccountsExternalComponent implements OnInit, OnDestroy {
       message: 'Zenodo credentials are used for creating Digital Object Identifiers (DOIs) on Zenodo.',
       show: false,
       logo: 'zenodo.jpg'
+    },
+    {
+      name: 'ORCID',
+      source: TokenSource.ORCID,
+      bold: '',
+      message: 'ORCID credentials are used for linking ORCID IDs to workflows.',
```

Comment:
I don't think this message is quite right, but on the other hand, we're not sure what level of support we're going to add for 1.9. Maybe for now `ORCID credentials are used for linking ORCID IDs to workflows published on Zenodo`?

Or maybe this is fine. Anybody else have an opinion?

Reply:
:man_shrugging:

Reply:
`ORCID credentials are used for linking ORCID IDs to workflows published on Zenodo` makes sense. I also saw the ticket on promoting it on user profiles (stargazers, organization membership) which makes sense to me